docs/software/remote_desktop.md

```diff
@@ -261,9 +261,65 @@ Once installed, available VirtualHere devices will be shown in the client user i
     The base installation of this DietPi software package installs the VirtualHere server. Basically it starts as a trial version supporting only one single USB device. To overcome this, you need to [buy a licence](https://virtualhere.com/purchase).
 
 !!! warning "USB Storage WARNING"
```

Comment:
Proposal for more clear warning:

```
!!! warning "USB Storage - WARNING: Data loss, service and system crashes may occur"
    USB devices cannot be used on the host server and the client system at the same time. VirtualHere will forcefully "detach" even actively used USB drives on the host system, once you start using them with the client. Be hence very careful to not select the wrong USB device in clients, especially when DietPi userdata or swap files are located on a USB drive.  
    If a device must stay available at the server, it is best to let it be ignored by VirtualHere, making use of the `IgnoredDevices` option: It takes `xxxx/yyyy` as value with `xxxx` being the vendor ID and `yyyy` being the device ID, which can be obtained from the output of `lsusb`. See the "Configuration" tab below and the official documentation link for further details.
```

Resulting appearance:
![image](https://user-images.githubusercontent.com/57687516/164545705-c5ee5c5b-e93a-4e8a-b2d1-69e7e26f215a.png)